orbit/cmd/orbit/orbit.go

```diff
@@ -1244,9 +1245,27 @@ func (d *desktopRunner) execute() error {
 		opts = append(opts, execuser.WithEnv("FLEET_DESKTOP_INSECURE", "1"))
 	}
 
+	var buf bytes.Buffer
+
 	for {
 		// First retry logic to start fleet-desktop.
 		if done := retry(30*time.Second, false, d.interruptCh, func() bool {
+			wcmd := exec.Command("/usr/bin/stat", "-f", "%Su", "/dev/console")
+			wcmd.Stderr = os.Stderr
+			wcmd.Stdout = &buf
+			if err := wcmd.Run(); err != nil {
```

Comment:
Instead of those separate steps and use of a buffer, you can use `exec.Command(...).Output()` to get the stdout output (presumably you don't care about stderr as it would only be for error output). It takes care of running the command and capturing the output in one go (https://pkg.go.dev/os/exec#Cmd.Output).

Reply:
Aha I didn't know about this method! Good idea! Question: do we care about the stderr output from the command line? The way it's done now, that output should go into the orbit log file directly. It could be helpful for debugging in case that command somehow fails, but not sure what the standard is in our codebase.

Reply:
If the command fails (and `cmd.Stderr` has not been set), the stderr will automatically be collected into the returned error when it is an `exec.ExitError` (https://pkg.go.dev/os/exec#ExitError). You can use that (via `errors.As(err, &exitErr)` to ensure it is of that type) to add the stderr output to the error log.

Reply:
Yep, saw that ExitError provided the stderr output. Sounds good!